src/js/claims-status/components/ClaimPhase.jsx

```diff
@@ -44,28 +44,28 @@ export default class ClaimPhase extends React.Component {
     switch (event.type) {
       case 'phase_entered':
         return (
-          <div className="claims-evidence-item columns usa-width-three-fourths medium-9">
+          <div className="claims-evidence-item">
```

Comment:
Is `claims-evidence-item` being used?

Reply:
Looks like not, aside from tests. I'll remove it.

Reply:
I changed my mind on this, the markup here makes writing the tests without this class a pain, so I'm going to leave it.